worker/appm/conversion/gateway.go

```diff
@@ -447,6 +449,15 @@ func (a *AppServiceBuild) createServiceAnnotations() map[string]string {
 	return annotations
 }
 
+func (a *AppServiceBuild) createKubernetesNativeService(port *model.TenantServicesPort) *corev1.Service {
+	svc := a.createInnerService(port)
+	svc.Name = port.K8sServiceName
+	if svc.Name == "" {
+		svc.Name = fmt.Sprintf("%s_%d", a.service.ServiceAlias, port.ContainerPort)
```

Comment:
For forward compatibility, is the default service name considered unchanged?

Reply:
The default name will not affect compatibility, right? There should be no direct use of service name before, but labels are used

Reply:
If there is a user communicating with the service name, or something else, it is not recommended to change this parameter unless it is particularly necessary.

Reply:
In the default mode, the service name is unchanged, only the format used in the Kubernetes native Service.

Reply:
Mainly because rainbond-console is not easy to get the default service_name without accessing the data center

Reply:
It is OK to only take effect in native.